### PR TITLE
Add basic insurance policy

### PR DESCRIPTION
**Pull request by Liang-Hsuan**

### CODE REVIEW

File: app/permissions.acl [hunk 1]
```diff
@@ -1,63 +1,110 @@
...
+rule InsuranceProviderUpdateOwnPolicy {
+    description: "Allow insurance providers to update their own policies"
+    participant(insurer): "org.example.mynetwork.InsuranceProvider"
+    operation: CREATE
+    resource(insurance): "org.example.mynetwork.UpdateInsurancePolicy"
+    condition: (insurance.insurance.insurer.getIdentifier() == insurer.getIdentifier())
+    action: ALLOW
+}
+
+// rule InsuranceProviderReadOwnTransactions {
```
Comment (Liang-Hsuan) [inline app/permissions.acl]: This will be done in the future

File: app/lib/logic.js [hunk 2]
```diff
@@ -1,43 +1,60 @@
...
-    await assetRegistry.update(tx.asset);
-
-    // Emit an event for the modified asset.
-    let event = getFactory().newEvent('org.example.mynetwork', 'SampleEvent');
-    event.asset = tx.asset;
-    event.oldValue = oldValue;
-    event.newValue = tx.newValue;
-    emit(event);
+async function updateInsurancePolicy(tx) { // eslint-disable-line no-unused-vars
+    const namespace = 'org.example.mynetwork';
```
Comment (Tan-He) [inline app/lib/logic.js]: unused var - namespace

File: app/permissions.acl (hunk 1 above)
Comment (Tan-He) [reply app/permissions.acl]: Create an issue?

Comment (Tan-He) [reply app/permissions.acl]: Reference this > InsuranceProviderReadTheirPatient

**Pull request merged**